src/server.c

```diff
@@ -5558,8 +5566,7 @@ static void sigShutdownHandler(int sig) {
         rdbRemoveTempFile(getpid(), 1);
         exit(1); /* Exit with an error since this was not a clean shutdown. */
     } else if (server.loading) {
-        serverLogFromHandler(LL_WARNING, "Received shutdown signal during loading, exiting now.");
-        exit(0);
+        serverLogFromHandler(LL_WARNING, "Received shutdown signal during loading, scheduling shutdown.");
```

Comment:
i think it'd be nice to log that we got it during loading in some way. not sure which is the cleanest.

Reply:
Maybe just consolidate that into the `msg` in the first place (i.e. append " during loading" if `server.loading`)?

Reply:
these strings are currently static, so it complicates things a bit, but i guess it's not that bad.
@perryitay can you handle that?

Reply:
The simplest way to preserve the old log behaviour would be to just assign msg here as in `msg = "Received shutdown signal during loading, scheduling shutdown.";` instead of logging directly.

Reply:
ok. it hides the type of the signal. but i'm ok with that.